pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -1151,3 +1174,126 @@ func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
 		},
 	}
 }
+
+func GetBootstrapClusterRoleBindings() []authorizationapi.ClusterRoleBinding {
+	openshiftClusterRoleBindings := GetOpenshiftBootstrapClusterRoleBindings()
+	kubeClusterRoleBindings, err := GetKubeBootstrapClusterRoleBindings()
+	// coder error
+	if err != nil {
+		panic(err)
+	}
+	kubeSAClusterRoleBindings, err := GetKubeControllerBootstrapClusterRoleBindings()
+	// coder error
+	if err != nil {
+		panic(err)
+	}
+
+	// openshift controllers and kube controllers have different prefixes
+	// so we only need to check conflicts on the "normal" cluster rolebindings
+	openshiftClusterRoleBindingNames := sets.NewString()
+	kubeClusterRoleBindingNames := sets.NewString()
+	for _, clusterRoleBinding := range openshiftClusterRoleBindings {
+		openshiftClusterRoleBindingNames.Insert(clusterRoleBinding.Name)
+	}
+	for _, clusterRoleBinding := range kubeClusterRoleBindings {
+		kubeClusterRoleBindingNames.Insert(clusterRoleBinding.Name)
+	}
+
+	conflictingNames := kubeClusterRoleBindingNames.Intersection(openshiftClusterRoleBindingNames)
+	extraRBACConflicts := conflictingNames.Difference(clusterRoleBindingConflicts)
+	extraWhitelistEntries := clusterRoleBindingConflicts.Difference(conflictingNames)
+	switch {
+	case len(extraRBACConflicts) > 0 && len(extraWhitelistEntries) > 0:
+		panic(fmt.Errorf("kube ClusterRoleBindings conflict with openshift ClusterRoleBindings: %v and ClusterRoleBinding whitelist contains a extraneous entries: %v ", extraRBACConflicts.List(), extraWhitelistEntries.List()))
+	case len(extraRBACConflicts) > 0:
+		panic(fmt.Errorf("kube ClusterRoleBindings conflict with openshift ClusterRoleBindings: %v", extraRBACConflicts.List()))
```

Comment:
> Any reason you chose the panic the server route instead of failing a unit test somewhere?

I've structured the code to allow both.  Turns out, this should fail the unit test creating the list of bootstrap roles, right?  Thought of that later.